pkg/kotsadm/types/constants.go

```diff
@@ -13,7 +13,7 @@ const ExcludeValue = "true"
 const BackupLabel = "kots.io/backup"
 const BackupLabelValue = "velero"
 
-const TroubleshootKey = "troubleshoot.io/kind"
+const TroubleshootKey = "troubleshoot.sh/kind"
```

Comment:
should we continue supporting the old label? it's not documented [here](https://troubleshoot.sh/docs/support-bundle/discover-cluster-specs/), but maybe someone started using it at some point?